WordPress/Classes/ViewRelated/Notifications/Tools/IntrinsicTableView.swift

```diff
@@ -11,7 +11,14 @@ import Foundation
 ///
 class IntrinsicTableView: UITableView
 {
+    override var contentSize:CGSize {
+        didSet {
+            self.invalidateIntrinsicContentSize()
+        }
+    }
+
     override func intrinsicContentSize() -> CGSize {
+        layoutIfNeeded()
```

Comment:
I guess this is OK though as the superview is really the one asking for this size, baed on the view's layout itself.